### PR TITLE
[WIP] Orb Concepts page rewrite

### PR DESCRIPTION
**Pull request by KyleTryon**
# Description
Reworked the content for the orb concepts page.

### CODE REVIEW

File: jekyll/_cci2/orb-concepts.md [hunk 1]
```diff
@@ -7,203 +7,137 @@ categories: [getting-started]
...
+    type: string
+```
 
-- environment variables to populate
-- which shell to use
-- what size resource_class to use
+For more information, see the guide to [Authoring Reusable Executors]({{site.baseurl}}/2.0/reusing-config/#authoring-reusable-executors) and the registry page for the [Node Orb](https://circleci.com/orbs/registry/orb/circleci/node#executors-default).
 
-When you declare an executor in a configuration outside of jobs, you can use these declarations for all jobs in the scope of that declaration, enabling you to reuse a single executor definition across multiple jobs.
+### Jobs
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-concepts.md:64]: Need to add in an example here

File: jekyll/_cci2/orb-concepts.md [hunk 2]
```diff
@@ -7,203 +7,137 @@ categories: [getting-started]
...
 
-All orbs are open, meaning that anyone can use them and see their source.
+For more information see our guide to [Authoring Reusable Commands]({{site.baseurl}}/2.0/reusing-config/#authoring-reusable-commands).
 
-### Design Methodology
+As an example, the AWS S3 orb includes a _command_ to copy a file or object to a new location. If your AWS authentication details are pre-stored as environment variables the syntax to use this command in your config is simply:
 
-Before using orbs, you may find it helpful to understand the various design decisions and methodologies that were used when these orbs were designed. Orbs were designed with the following considerations:
-
-* Orbs are transparent - If you can execute an orb, you and anyone else can view the source of that orb.
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-concepts.md:24]: Is this list no longer correct, @KyleTryon?

File: jekyll/_cci2/orb-concepts.md [hunk 3]
```diff
@@ -7,203 +7,137 @@ categories: [getting-started]
...
 
-- docker, machine, or macos
-- environment
-- working_directory
-- shell
-- resource_class
+## Namespaces
 
-The example below shows a simple example of using an executor:
+A _namespace_ is a unique identifier claimed by a user or organization to group a set of orbs by author. Each user and organization may claim _one_ unique and immutable namespace name. Each namespace my contain main uniquely named orbs.
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-concepts.md:72]:
```suggestion
A _namespace_ is a unique identifier claimed by a user or organization to group a set of orbs by author. Each user or organization can claim _one_ unique and immutable namespace. Each namespace can contain many uniquely named orbs.
```

File: jekyll/_cci2/orb-concepts.md [hunk 4]
```diff
@@ -7,203 +7,137 @@ categories: [getting-started]
...
-      - image: circleci/ruby:2.4.0
+For example, the `circleci/rails` orb may coexist in the registry with an orb called `<username>/rails` because they are in separate namespaces.
 
-jobs:
-  my-job:
-    executor: my-executor
-    steps:
-      - run: echo outside the executor
-```
+Organizations are, by default, limited to claiming only one namespace. This policy is designed to limit name-squatting and namespace noise. If you are in need of changing your namespace, please contact your account team at CircleCI.
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-concepts.md:76]:
```suggestion
Organizations are, by default, limited to claiming only one namespace. This policy is designed to limit name-squatting and namespace noise. If you need to change your namespace, please contact your account team at CircleCI.
```

File: jekyll/_cci2/orb-concepts.md [hunk 5]
```diff
@@ -7,203 +7,137 @@ categories: [getting-started]
...
-jobs:
-  my-job:
-    executor: my-executor
-    steps:
-      - run: echo outside the executor
-```
+Organizations are, by default, limited to claiming only one namespace. This policy is designed to limit name-squatting and namespace noise. If you need to change your namespace, please contact your account team at CircleCI.
 
-Notice in the above example that the executor `my-executor` is passed as the single value of the key executor. Alternatively, you can pass `my-executor` as the value of a name key under executor. This method is primarily employed when passing parameters to executor invocations. An example of this method is shown in the example below.
+By default, any created namespaces will appear as a "community" namespace on the [Orb Registry](https://circleci.com/orbs/registry/).
```
Comment (rosieyohannan) [inline jekyll/_cci2/orb-concepts.md:78]:
```suggestion
By default, created namespaces appear as a "community" namespace in the [Orb Registry](https://circleci.com/orbs/registry/).
```

File: jekyll/_cci2/orb-concepts.md (hunk 2 above)
Comment (KyleTryon) [reply jekyll/_cci2/orb-concepts.md:24]: The Design Methodology section? This seemed to cover some redundant topics, such as semver and immutability (although maybe a quick note on immutability do that). Same with the piece on organization admins, that is covered elsewhere and should probably be touched on again in the development page but seemed redundant here.

will send over a small change for the immutability piece

File: jekyll/_cci2/orb-concepts.md (hunk 1 above)
Comment (KyleTryon) [reply jekyll/_cci2/orb-concepts.md:64]: @rosieyohannan I will add it here but I was thinking we could use the same example we are using for the orb intro page.

File: jekyll/_cci2/orb-concepts.md (hunk 2 above)
Comment (KyleTryon) [reply jekyll/_cci2/orb-concepts.md:24]: Expanded on the orb versions section, I think this addresses this.

Comment (rosieyohannan) [reply jekyll/_cci2/orb-concepts.md:24]: Brilliant, thanks!

File: jekyll/_cci2/orb-concepts.md (hunk 1 above)
Comment (rosieyohannan) [reply jekyll/_cci2/orb-concepts.md:64]: Oh yeah good idea

**Pull request merged**